config/v1/types_apiserver.go

```diff
@@ -30,6 +30,12 @@ type APIServerSpec struct {
 	// - ConfigMap.Data["ca-bundle.crt"] - CA bundle.
 	// +optional
 	ClientCA ConfigMapNameReference `json:"clientCA"`
+	// extraCORSAllowedOrigins lists additional, user-defined regular expressions describing hosts for which the
+	// API server allows access using the CORS headers. This is needed to access the API and the integrated OAuth
+	// server from JavaScript applications.
+	// The values are regular expressions that correspond to the Golang regular expression language.
+	// +optional
+	ExtraCORSAllowedOrigins []string `json:extraCORSAllowedOrigins,omitempty`
```

Comment:
This needs to be consistent.  It's either `corsAllowedOrigins` or `AdditionalCORSAllowedOrigins` to match the usage in images.config.openshift.io and builds.config.openshift.io for extra values.

Reply:
Additional sounds good.